app/controllers/api/v1/application_controller.rb

```diff
@@ -0,0 +1,6 @@
+module Api
+    module V1 
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/blyscuit/rails-example-one/blob/c919a05a12101ae07b4c21bf82ccf33697056e9e/app/controllers/api/v1/application_controller.rb#L2"></span>Use 2 (not 4) spaces for indentation.</td>
    </tr>
  </tbody>
</table>